### PR TITLE
Add Game Events Reference

### PR DESCRIPTION
**Pull request by CommanderFoo**
First draft of the game events reference.

### CODE REVIEW

File: src/references/game_events.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+title: Game Events Reference
+tags:
+    - Reference
+---
+
+# Game Events
+
+## Overview
+
+Events are designed to help increase Engagement and Retention in your games. Users will be able to browse Events for all games or view Events specific to games on the game details page.
```
Comment (coreslinkous) [inline src/references/game_events.md]: lowercase "engagement" and "retention"
Events -> **Game Events**

Events *for specific games (or "of" "from" "created for",  your choice)

I wonder if we can make sure people know what we mean by the game details page. It isn't explicitly called that anywhere. Picture might be best, added once we have some live games.


Everywhere: If you are looking for the proper noun, system-made-by-Core, call it "Game Events" (only needs to be bold once per paragraph max) and then referring to the general concept of an event in a game, make it lowercase: "events".

File: src/references/game_events.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+On the **Events** page, click on the button **Create Event**, which will take you to the **Create Event** form. All the fields will need to be filled in for the event.
+
+![!Create Event Button](../img/GameEvents/create_event_button.png){: .center loading="lazy" }
+
+### Events Per Game Limit
+
+When a game is selected for the event, the drop down will display to the right of the game, how many events have been created, and the maximum amount that can be created for each game.
+
+For creators in the Perk Program, 15 slots are available per game, and 5 slots available to creators not in the Perks Program. Slots will be freed up when events have finished, or been cancelled.
```
Comment (coreslinkous) [inline src/references/game_events.md]: something besides "slots" unless it says that somewhere that I couldn't find. "current and future events" the first time and "total events" the second.

File: src/references/game_events.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+| **Cancel Event** | Cancels the event. A prompt will show to get confirmation. An event that is scheduled, or active can be cancelled at any time. |
+| **Copy Game Event Id** | Copies the unique game event Id for this event, which can be used in the editor with the **UI Event RSVP Button**. This Id contains the game Id and Event Id separated by a hyphen (-). |
+| **Copy Event Link** | Copies the URL to the event page for the game. |
+| **View Event Details** | Opens up the event page for the game. |
+
+![!Options Menu](../img/GameEvents/options_menu.png){: .center loading="lazy" }
+
+## Registering for an Event
+
+Players can register for events from the [Events](https://coregames.com/events) page, or from inside **Core**. Players registering for an event will allow them keep track of specific events they are interested in, and receive notifications when events begin.
```
Comment (coreslinkous) [inline src/references/game_events.md]: - put "page" inside of the link: "Events page" (it's just a screenreader thing)
- "or from inside Core" -> from the Core client, or in the games themselves if creators implement a UI RSVP Button.

File: src/references/game_events.md [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+Each game event created will have a **Game Event Id** that can be copied and added to the **Event ID** property of the **UI Event RSVP Button** in the **Properties Panel**. If no ID is entered, or is invalid, then the button will be disabled.
+
+![!Register For Event](../img/GameEvents/event_button.png){: .center loading="lazy" }
+
+![!Register For Event](../img/GameEvents/event_id_property.png){: .center loading="lazy" }
+
+## Game Events API
+
+There is an API that can be accessed from Lua scripts that can retrieve information about events. Information such as event name, description, amount of players registered, start and end dates are available.
```
Comment (coreslinkous) [inline src/references/game_events.md]: Change "There is an API" to "The GameEvents API" or whatever the class is actually called. And make it a link to the API

Comment (coreslinkous) [reply src/references/game_events.md]: CoreGameEvent

File: src/references/game_events.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+---
+
+# Game Events
+
+## Overview
+
+Events are designed to help increase Engagement and Retention in your games. Users will be able to browse Events for all games or view Events specific to games on the game details page.
+
+Events are available to everyone, however, Creators in the Perks Program will have the ability to create more events, and events with a longer length.
+
```
Comment (coreslinkous) [inline src/references/game_events.md:20]: Add a sentence here to clarify that this is not about Broadcast Events in Lua and link to that page.

**Review by coreslinkous — CHANGES_REQUESTED** [review]
Just a couple things for clarity

File: src/references/game_events.md (hunk 1 above)
Comment (CommanderFoo) [reply src/references/game_events.md]: Yeah, adding a picture would be a good idea. Will make a note of it.

**Pull request merged**